### PR TITLE
CSS tweaks, filename save, add GCP URL variable, add GA script

### PR DESCRIPTION
**Pull request by SimonGrn**
- add handling of environment variable for GCP URL (since it's used in 2 places now)
- add storing of the filename to keep the creation date in DB
- add a direct link to the build when available (only last 2 weeks on GCP)
- minor CSS tweaks for the home table
- add Google Analytics script
- change 'download' title in home table

### CODE REVIEW

File: application/controllers/Home.php [hunk 1]
```diff
@@ -6,6 +6,21 @@ public function index()
         $this->load->model('Execution');
         $this->load->helper('my_duration');
 
+        //get all data from GCP
+        $GCP_files_list = [];
+        $url = getenv('QANB_GCPURL') !== false ? getenv('QANB_GCPURL') : 'https://storage.googleapis.com/prestashop-core-nightly/';
```
Comment (PierreRambaud) [inline application/controllers/Home.php]:
```suggestion
        $url = getenv('QANB_GCPURL') ?: 'https://storage.googleapis.com/prestashop-core-nightly/';
```

File: application/controllers/Hook.php [hunk 2]
```diff
@@ -13,9 +13,7 @@ public function add()
         $this->load->model('Test');
 
         //is there a GCP URL in environment variable ?
-    if (getenv('QANB_GCPURL') !== false && strpos(getenv('QANB_GCPURL'), 'http') !== false) {
-            $this->GCPURL = getenv('QANB_GCPURL');
-        }
+        $this->GCPURL = getenv('QANB_GCPURL') !== false ? getenv('QANB_GCPURL').'reports/' : 'https://storage.googleapis.com/prestashop-core-nightly/reports/';
```
Comment (PierreRambaud) [inline application/controllers/Hook.php]: Duplicate from Home.php

File: application/views/home.php [hunk 3]
```diff
@@ -51,13 +52,31 @@
...
+                                        foreach($gcp_files_list as $gcp_file) {
+                                            preg_match($pattern, $gcp_file, $matches);
+                                            if (isset($matches[1]) && $matches[1] != '') {
+                                                $branch = $matches[1];
+                                                $download_link = '<a href="https://storage.googleapis.com/prestashop-core-nightly/'.$gcp_file.'"><i class="material-icons">cloud_download</i> Download build</a>';
+                                                break;
+                                            }
+                                        }
+                                    }
+                                    unset($matches_filename);
```
Comment (PierreRambaud) [inline application/views/home.php]: No need to unset.

File: application/views/templates/footer.php [hunk 4]
```diff
@@ -1,2 +1,11 @@
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-2753771-46"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'UA-2753771-46');
```
Comment (PierreRambaud) [inline application/views/templates/footer.php]: make it configurable.

File: application/controllers/Home.php (hunk 1 above)
Comment (SimonGrn) [reply application/controllers/Home.php]: I genuinely didn't know this syntax, thanks

File: application/views/home.php (hunk 3 above)
Comment (SimonGrn) [reply application/views/home.php]: Done

File: application/views/templates/footer.php (hunk 4 above)
Comment (SimonGrn) [reply application/views/templates/footer.php]: If I want to make it configurable, that means another environment variable.

File: application/controllers/Hook.php (hunk 2 above)
Comment (SimonGrn) [reply application/controllers/Hook.php]: So I changed it and now it's a config value, which can be overwritten through env variables.

**Pull request merged**